oraclizeAPI_0.4.25.sol

```diff
@@ -285,9 +335,9 @@ End solidity-cborutils
  */
 
 contract usingOraclize {
-    uint constant day = 60*60*24;
-    uint constant week = 60*60*24*7;
-    uint constant month = 60*60*24*30;
+    uint constant day = 60 * 60 * 24;
+    uint constant week = 60 * 60 * 24 * 7;
+    uint constant month = 60 * 60 * 24 * 30;
```

Comment:
Ibid comment above re stylistic changes to `0.4.25` w/r/t operator spacing ∴ won't fix.

Re `month_`, I don't see a gain from adding more weight to the API on deploy by adding more constants here since they exist for legacy/backwards-compatibility reasons.